jenkins/map.jinja

```diff
@@ -5,6 +5,14 @@
         'group': 'jenkins',
         'home': '/var/lib/jenkins',
         'port': '80',
-        'server_name': None
+        'server_name': None,
```

Comment:
Would you like to run a build with some find / -name jenkins-cli.jar just
for curiosity ?

Note: you can run a build by pushing a new commit on your branch and the PR
will have updated statuses and links to log like above.